src/com/mateacademy/calculator/CalculatorRealisation.java

```diff
@@ -0,0 +1,41 @@
+package com.mateacademy.calculator;
+
+public class CalculatorRealisation {
+    Calculator calculator = (firstNumber, secondNumber, operation) -> {
+        return (operation == '+') ? addition(firstNumber, secondNumber)
```

Comment:
It is not a good idea to use so many ternary operators in one equation. 
Please use Map to solve this problem.
Also, you should use a lambda function to solve this task.